### PR TITLE
Add Rowley Biaryl torsiondrive set.

### PR DESCRIPTION
**Pull request by jthorton**
I have processed the pdbs in the Rowley biaryl set ready for submission for torsiondriving. The last block of the notebook `QCSubmit workflow.ipynb` should be able to submit the dataset to the client. 

I can remove the PDBs if required as I have added a smiles file of the inputs as well.

### CODE REVIEW

**Comment by trevorgokey** [comment]
Thanks Josh. Looking forward to trying out QCSubmit. Few notes:

1. Please add Mayer bond orders as a keyword to this and future OpenFF submissions.
2. I was not able to reproduce your code in the ipynb. Specifically, `ImportError: cannot import name 'TorsionIndexer' from 'qcsubmit.common_structures'`. I just cloned QCSubmit from master. Were you using a branch?
3. Loading the json directly:
```
 biaryl_dataset = TorsiondriveDataset.parse_file("biaryl_dataset.json")                                           
---------------------------------------------------------------------------
ValidationError                           Traceback (most recent call last)
<ipython-input-15-16374bd72a00> in <module>
----> 1 biaryl_dataset = TorsiondriveDataset.parse_file("biaryl_dataset.json")

~/.local/miniconda3/envs/oFF/lib/python3.7/site-packages/pydantic/main.cpython-37m-x86_64-linux-gnu.so in pydantic.main.BaseModel.parse_file()

~/.local/miniconda3/envs/oFF/lib/python3.7/site-packages/pydantic/main.cpython-37m-x86_64-linux-gnu.so in pydantic.main.BaseModel.parse_obj()

~/.local/miniconda3/envs/oFF/lib/python3.7/site-packages/qcsubmit/datasets.py in __init__(self, **kwargs)
    307         """
    308 
--> 309         super().__init__(**kwargs)
    310 
    311         # set the collection type here

~/.local/miniconda3/envs/oFF/lib/python3.7/site-packages/pydantic/main.cpython-37m-x86_64-linux-gnu.so in pydantic.main.BaseModel.__init__()

ValidationError: 3 validation errors for TorsiondriveDataset
filtered_molecules -> StandardConformerGenerator
  'NoneType' object is not iterable (type=type_error)
filtered_molecules -> LinearTorsionRemoval
  'NoneType' object is not iterable (type=type_error)
filtered_molecules -> UnconnectedTorsionRemoval
  'NoneType' object is not iterable (type=type_error)
```

**Comment by jthorton** [comment]
Thanks for giving it a look over @trevorgokey !

Mayer indices added I have also opened a [PR](https://github.com/openforcefield/qcsubmit/issues/22) into QCSubmit to make sure this is now a default setting on all future submissions. 

Good catch I was on the `torsion_selection` branch Ill try and get this merged before the meeting which should clear up these issues.

**Comment by dotsdl** [comment]
Awesome work @jthorton! This is looking slick! I love the reporting; this adds to our log for the PR showing status over time!

File: management/validation.py [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+    return [dataset_data.get("dataset_name", missing),
+            dataset_data.get("dataset_type", missing),
+            dataset_data.get("method", missing),
+            dataset_data.get("basis", missing)]
+
+
+def main_validation(dataset_names):
+    """
+    Generate a report dataframe for each dataset found.
+    """
```
[the remark this answers is not in the document]
Comment (jthorton) [reply management/validation.py:132]: Good point it is something I am a little worried about as well. I think ill try what you suggest to keep the ordering consistent and then later we can look at ways to make extension a bit safer.

**Comment by jthorton** [comment]
Looks like we are picking up the errors correctly now.

**Comment by dotsdl** [comment]
We've got one `OptimizationRecord` INCOMPLETE that may be hanging. Our COMPLETE and INCOMPLETE `OptimizationRecord` values have remained the same for at least 14 hours. Will check tomorrow if this is still the case, then grab the offending record for details.

**Comment by dotsdl** [comment]
This is the INCOMPLETE `OptimizationRecord`:

```
{'id': '21420672',
 'hash_index': '8872e70f24008ab46368f61fe2edb534033c4e58',
 'procedure': 'optimization',
 'program': 'geometric',
 'version': 1,
 'protocols': {},
 'extras': {},
 'stdout': '24827288',
 'stderr': None,
 'error': None,
 'task_id': None,
 'manager_name': None,
 'status': <RecordStatusEnum.incomplete: 'INCOMPLETE'>,
 'modified_on': datetime.datetime(2020, 7, 27, 12, 35, 38, 610329),
 'created_on': datetime.datetime(2020, 7, 27, 12, 35, 38, 610330),
 'provenance': {'creator': 'geomeTRIC',
  'version': '0.9.7.2',
  'routine': 'geometric.run_json.geometric_run_json',
  'cpu': 'Intel(R) Xeon(R) Gold 6230 CPU @ 2.10GHz',
  'username': 'qcfractal',
  'qcengine_version': 'v0.15.0',
  'wall_time': 122.74065685272217,
  'hostname': 'openff-qca2-755bd6d8cb-562xv'},
 'schema_version': 1,
 'initial_molecule': '14915990',
 'qc_spec': {'driver': <DriverEnum.gradient: 'gradient'>,
  'method': 'b3lyp-d3bj',
  'basis': 'dzvp',
  'keywords': '2',
  'program': 'psi4'},
 'keywords': {'coordsys': 'tric',
  'enforce': 0.1,
  'epsilon': 0,
  'reset': True,
  'qccnv': True,
  'molcnv': False,
  'check': 0,
  'trust': 0.1,
  'tmax': 0.3,
  'maxiter': 300,
  'convergence_set': 'gau',
  'constraints': {'set': [{'type': 'dihedral',
     'indices': [7, 3, 10, 9],
     'value': -150}]},
  'program': 'psi4'},
 'energies': [-489.40847967885725,
  -489.4095300428588,
  -489.4101609059971,
  -489.4109301491452,
  -489.41093188494636,
  -489.4109591137581,
  -489.41095877272556,
  -489.4109755521607,
  -489.41097619829986],
 'final_molecule': '14916050',
 'trajectory': ['21420643',
  '21420701',
  '21420702',
  '21420703',
  '21420704',
  '21420705',
  '21420706',
  '21420707',
  '21420708']}
```

@bennybp does this fit the profile of one of those INCOMPLETEs we've been seeing due to spec changes, or could this be due to an interrupted transaction on the database (inconsistent state due to non-atomic operation)? This is a recent submission, so the former seems not possible here.

**Comment by dotsdl** [comment]
Moved to "Requires Scientific Review"; this is a technical issue with QCArchive, not a scientific issue. We will address and move to complete when resolved.

**Comment by dotsdl** [comment]
Returned to Error Cycling to see if remaining INCOMPLETE clears. We are seeing what looks like clearing on #121, though not obvious how at this time.

**Comment by dotsdl** [comment]
Moved to "Requires Scientific Review" as this submission is no longer advancing. It's likely that we can clear the final INCOMPLETE with work currently in progress on the public QCArchive server. We will park it here until we can test that functionality.

**Pull request merged**